docs/editors.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: Editors
+description: Learn how to use Babel in your editor of choice
+permalink: /docs/editors
+---
+
+## Syntax Highlighting
+
+These days, many popular editors support ES2015+ syntax highlighting
```

Comment:
Absolutely. I can add that. If you happen to have such a picture with various ES syntax constructs in it, let me know. Otherwise, I'll create it myself.